.github/workflows/release-prod.yml

```diff
@@ -2,21 +2,21 @@ name: Release Production App
 
 on:
   release:
-    types: [created]
+    types: [ created ]
 
 jobs:
-  build_and_test:
+  build:
     uses: ./.github/workflows/build.yml
     with:
-      storeArtifacts: true
-      prodRelease: true
-      tagName: ${{ github.event.release.tag_name }}
+      store_artifacts: true
+      prod_release: true
+      tag_name: ${{ github.event.release.tag_name }}
     secrets: inherit
 
-  prod_release_ios:
+  release_ios:
     name: Upload iOS build to App Store connect
     runs-on: macos-latest
-    needs: [build_and_test]
```

Comment:
We should not release the Production App if tests are failing, thus we need a dependency on tests

Reply:
For the same reason above i do not believe this will be relevant